### DISCUSSION TITLE
Pls add an dev branch

### DISCUSSION THREAD

**Discussion by jpx40** [prasanthrangan/hyprdots#1176] [General]
Hey,
 I want to push the first version of the new helper CLI that i currently make. But want to test this with you on a dev branch. we will brake lots of stuff so yeah

**Comment by ghost** [comment]
> I want to push the first version of the new helper CLI that i currently make.

A new helper cli? Isn't that just hyprdots-ctl or are you trying to do something different?

**Reply by jpx40** [reply]
more of a install menu maybe more.

**Comment by prasanthrangan** [comment]
Hi, please check [Hyprdots-ctl](https://github.com/kRHYME7/Hyprdots-ctl)  its already available... so you can directly contribute there.

Regarding `dev` branch, let me create one (`hyde`) ... I think its necessary as this project gets bigger.

**Reply by jpx40** [reply]
i don't know i mean it's just an installer/ isntall menu what i did

**Reply by dieBakterie** [reply]
It's for ricing and configs for vim/nvim/neovim etc IT should be implemented there. It's a good place for it.

**Comment by se7en-x230** [comment]
Can I use the dev branch as my main update branch and if how?

**Reply by kRHYME7** [reply]
Yeah, In case you have issues, open it in the discussion panel as this branch is not yet ready. BTW, Here's the discord server https://discord.gg/Jeq7TGKP  Not done yet but would love to receive suggestions to make it better.

**Reply by se7en-x230** [reply]
How would I clone this branch?

git branch -a 
Only showed main yesterday

**Reply by jpx40** [reply]
@se7en-x230 
u use "git switch hyde" to swtich the branch
https://git-scm.com/docs/git-switch

**Reply by kRHYME7** [reply]
@se7en-x230  I just want to confirm if ``` Hyprdots branch ``` works ?

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/4b92d88a-d3d5-4096-b7f2-477e177393ed)

**Reply by jpx40** [reply]
what git cli ist this

**Reply by se7en-x230** [reply]
Yes 

![Screenshot_20240331-062536](https://github.com/prasanthrangan/hyprdots/assets/69777713/0c04d3af-4770-4015-a64e-72f66a09b7be)
